src/main/java/uk/gov/hmcts/probate/services/submit/clients/MailClient.java

```diff
@@ -30,7 +30,7 @@ public String execute(JsonNode submitData, JsonNode registryData,  Calendar subm
         try {
             MimeMessage message = mailMessageBuilder.buildMessage(submitData, registryData, mailSender.getJavaMailProperties(), submissionTimestamp);
             mailSender.send(message);
-            return submitData.at("/submitdata/submissionReference").asText();
+            return submitData.at("").asText();
```

Comment:
Minor point especially as Jack's ticket removes this code entirely but it would be better to make the method have a void return or at least return true (as an indicator of success) instead of returning an empty string.

Reply:
I decided to do this because Jack's ticket was going to delete this. But yes, it can return a true.